code/citadel/organs/genitals.dm

```diff
@@ -198,21 +208,28 @@
 	if(H.is_groin_exposed(worn_stuff))
 		if(H.has_penis())
 			genitals_to_add += H.getorganslot("penis")
-		if(H.has_vagina())
-			genitals_to_add += H.getorganslot("vagina")
+		else
+			if(H.has_vagina()) //Sadly, a penis would cover the vagina and they use the same layer, so don't draw both.
```

Comment:
Overlays applied first will be on the bottom. So do vagina -> testicles(when we do get sprite support) -> penis